### DISCUSSION TITLE
Add multiple levels of headers from markdown to sidebar

### DISCUSSION THREAD

**Discussion by SebastianBienert** [scalar/scalar#905] [Ideas]
Currently adding markdown headers in spec description results in adding additional items in the sidebar (example spec: https://github.com/outline/openapi/blob/main/spec3.json). However, only one level (the highest which appears in the markdown) of heading is added. It looks like this is made by design: [source](https://github.com/scalar/scalar/blob/584849b5de65fffe123bed2b33d3a53c8ca7dc2b/packages/api-reference/src/components/Content/Introduction/Description.vue#L23C1-L25C66). I think it would be great if we can add at least one more level of heading so it would match the structure of endpoints documentation, where for each tag we can have collapsible section:
<img width="413" alt="image" src="https://github.com/scalar/scalar/assets/17458785/11c5c46f-a53b-4f88-97a0-1f0b5e520f86">

**Comment by hanspagel** [comment]
Love it! Let’s add this. Created an issue #906
